pkg/daemon/spawn.go

```diff
@@ -69,11 +70,17 @@ func Spawn(cfg *SpawnConfig) error {
 		"-bin", binPath,
 		"-db", dbPath,
 		"-sock", sockPath,
-		"-logprefix", logPathPrefix,
 	}
 
-	// TODO Redirect daemon stdout and stderr
+	daemonStdout, err := fsutil.ClaimFile(runDir, "daemon-*.log")
+	if err != nil {
+		return err
+	}
+	defer daemonStdout.Close()
+
+	procattrs := procAttrForSpawn(daemonStdout)
+	_, err = os.StartProcess(binPath, args, procattrs)
+	procattrs.Files[0].Close() // drop our reference to the /dev/null handle passed to the daemon
```

Comment:
Instead of letting `procAttrForSpawn` open `/dev/null` or `NUL` and relying on the implicit fact that `procattrs.Files[0]` is that file, it's better to open that file here and pass it as an argument to `procAttrForSpawn`; `os.DevNull` can be used as the filename.